modules/decline/test/src/smithy4s/decline/core/commonsSpec.scala

```diff
@@ -0,0 +1,14 @@
+package smithy4s.decline.core
+
+import weaver._
+
+object commonsSpec extends FunSuite {
```

Comment:
nitpick: in tests, we usually name objects starting with an uppercase letter... although the object being tested is `commons`, I would probably rename this to match other suites

Reply:
And in a PR about camel-case no less.